doc/syntax.rst

```diff
@@ -221,29 +245,28 @@ columns separated by commas.
 Quantum program
 ---------------
 
-The ``with`` statement indicates the device to run the program on,
-as well as providing device-specific options.
+The ``|`` operator is used to apply intrinsic quantum operations to quantum registers.
 
-Inside the indented ``with`` block, all operations are queued
-to be executed on the device, in the order they appear.
+All operations are queued to be executed on the device, in the order they appear.
```

Comment:
Is this something that needs to be said in the blackbird docs? i.e., is it more appropriate elsewhere?

Reply:
No, I can remove this.
```suggestion
```

Reply:
:+1: